src/components/contextual-menu-submenu/readme.md

```diff
@@ -0,0 +1,7 @@
+# gxg-menu
+
```

Comment:
I added all the descriptions on contextual-menu/readme.md, because that is where the story resides.
If I add comments here they wouldn´t be visible on the story.